presto-orc/src/main/java/com/facebook/presto/orc/reader/ListStreamReader.java

```diff
@@ -120,6 +120,10 @@ public Block readBlock(Type type)
             offsets[i] += offsets[i - 1];
         }
 
+        if (type.getTypeParameters().size() == 0) {
```

Comment:
Are we sure this is the right fix? Having no type parameters looks like a bug in some other part of Presto, as it seems like it never should have got here